### PR TITLE
Make psammead-styles treeshakable

### LINKED ISSUE #1688: Make psammead-styles treeshakable
**Is your feature request related to a problem? Please describe.**
When I made all psammead packages tree shakable I somehow missed psammead styles, my bad

**Describe the solution you'd like**
Add ESM exports to psammead styles in a non breaking way

**Testing notes**
Regression required.

Dev insight: Same as other packages

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Resolves #1688 

**Overall change:** _Restructures package to use nested `package.json`s to expose ESM code._

**Code changes:**

- _Restructures package to use nested `package.json`s to expose ESM code._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by thekp — CHANGES_REQUESTED** [review]
fix linting

**Review by thekp — APPROVED** [review]

File: packages/utilities/psammead-styles/colours/package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@bbc/psammead-styles/colours",
+  "private": "true",
```
Comment (andrew-nowak) [inline packages/utilities/psammead-styles/colours/package.json]:
```suggestion
  "private": true,
```

File: packages/utilities/psammead-styles/detection/package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@bbc/psammead-styles/detection",
+  "private": "true",
```
Comment (andrew-nowak) [inline packages/utilities/psammead-styles/detection/package.json]:
```suggestion
  "private": true,
```

File: packages/utilities/psammead-styles/font-styles/package.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@bbc/psammead-styles/font-styles",
+  "private": "true",
```
Comment (andrew-nowak) [inline packages/utilities/psammead-styles/font-styles/package.json]:
```suggestion
  "private": true,
```

File: packages/utilities/psammead-styles/fonts/package.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@bbc/psammead-styles/fonts",
+  "private": "true",
```
Comment (andrew-nowak) [inline packages/utilities/psammead-styles/fonts/package.json]:
```suggestion
  "private": true,
```

**Review by dr3 — COMMENTED** [review]
LGTM after @aacn500's suggestion :)

**Review by dr3 — APPROVED** [review]

**Comment by benjaminhobbs** [comment]
@dependabot rebase

**Pull request merged**